content/en/continuous_integration/setup_pipelines/jenkins.md

```diff
@@ -41,15 +44,94 @@ Install and enable the [Datadog Jenkins plugin][3] v3.1.0 or newer:
 7. (Optional) Configure your CI Instance name.
 8. Check the connectivity with the Datadog Agent.
 9. Save your configuration.
-10. To verify that CI Visibility is enabled, go to `Jenkins Log` and search for:
+
+{{< img src="ci/ci-jenkins-plugin-config.png" alt="Datadog Plugin configuration for Jenkins" style="width:100%;">}}
+{{% /tab %}}
+{{% tab "Using configuration-as-code" %}}
+If your Jenkins instance uses the [Jenkins `configuration-as-code` plugin][1], follow the next instructions:
```

Comment:
Opinionated, but I feel like the link here is too long. Maybe just link "configuration-as-code" or "configuration-as-code plugin"?

Reply:
changed